main.css

```diff
@@ -386,17 +365,268 @@ form > textarea:focus {
 
 .footer--section {
   width: 375px;
+  border-top: solid 1px white;
   display: flex;
   flex-direction: column;
   align-items: center;
 }
 
-.supporting-item-container:nth-child(2) {
-  margin: 25px 0 15px 0;
-}
-
 .footer--email {
+  font-size: 2rem;
   text-decoration: none;
   color: #fff;
-  margin-bottom: 25px;
+  margin-top: -2.5rem;
+  margin-bottom: 2.5rem;
+}
+
+@media screen and (min-width: 768px) {
+  header {
+    position: fixed;
+    top: 0;
+    z-index: 99;
+  }
```

Comment:
[Optional]
- [ ] The sticky navbar is not necessary for this project. But if you have it, you should give it a background color. It overlays the content and makes them unreadable.
![screeb](https://user-images.githubusercontent.com/1642014/121608127-fd917300-ca59-11eb-9ebc-a0c1cd65ffef.png)